daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Runner.scala

```diff
@@ -377,12 +377,9 @@ private[lf] class Runner(
       esf: ExecutionSequencerFactory,
       mat: Materializer,
   ): (Speedy.Machine, Future[SValue]) = {
-    val machine =
-      // TODO: https://github.com/digital-asset/daml/issues/12208
-      //  plug the logging context properly in Daml-script
-      LoggingContext.newLoggingContext(
-        Speedy.Machine.fromPureSExpr(extendedCompiledPackages, script.expr, traceLog, warningLog)(_)
-      )
+    val machine = LoggingContext.newLoggingContext(
```

Comment:
Factorizing the logging context in daml script is much more involved than in the scenario service. 
To do it in the clean way, we have to propagate it everywhere (in all the methods that already take already `implicit ec: ExecutionContext, mat: Materializer` (around 100 places)  + most of the method of the `Converter` + some extra other place). 
I am not sure it is worthwhile since we do not really care of the logging.

Reply:
That makes sense. How about that instead of creating it once and passing it around everywhere, we add a method that produces the empty context and then document why we’re fine with this? That way we have a single location to place the comment and if we do ever want to change it we can easily find all reference to that method and we don’t end up having to pass it around everywhere, we can just call the method.

Reply:
87e8a3922339acae95d15e8443dc9a1b06d3c8e2